content/tutorials/managing-config-and-secrets/index.md

```diff
@@ -0,0 +1,194 @@
+estimated_time: 10
+
+# # An optional list of collections this tutorial should be belong to. Collections are defined in data/tutorials/collections.yaml.
+# collections:
+#     - some-non-existent-collection
+---
+
+## Create a new project
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/). Next, create a new project by running the `pulumi new <language>` command, making sure to replace `<language>` with the supported language of your choice:
+
+```bash
+# example using python
+$ pulumi new python
+This command will walk you through creating a new Pulumi project.
+
+Enter a value or leave blank to accept the (default), and press <ENTER>.
+Press ^C at any time to quit.
+
+project name (pulumi-dev):  
+project description (A minimal Python Pulumi program):  
+Created project 'pulumi-dev'
+
+Please enter your desired stack name.
+To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`).
+stack name (dev): pulumi/dev
+Created stack 'dev'
+
+Installing dependencies...
+
+Creating virtual environment...
+Finished creating virtual environment
+Updating pip, setuptools, and wheel in virtual environment...
+...
+...
+Finished installing dependencies
+
+Your new project is ready to go!
+
+To perform an initial deployment, run `pulumi up`
+```
+
+This will create simple Pulumi program without any resources or configuration details.
+
+## Create configuration values
+
+In a Pulumi project, you can locally [store and retrieve configuration values](/docs/concepts/config/) using the `pulumi config set <key> [value]` command. To demonstrate, run the following command to create a configuration value with a key of `myEnvironment` and a value of `development`:
+
+```bash
+$ pulumi config set myEnvironment development
+```
+
+This value will be stored in your project's stack settings file `Pulumi.<your-stack-name>.yaml` as shown below:
+
+```yaml
+# Contents of Pulumi.<your-stack-name>.yaml file
+config:
+  pulumi-dev:myEnvironment: development
+```
+
+You can also list the configuration values for your stack in the command line. To do so, run the `pulumi config` command:
+
+```bash
+$ pulumi config
+
+KEY            VALUE
+myEnvironment  development
+pulumi:tags    {"pulumi:template":"python"}
+```
+
+To retrieve a specific configuration value, run the `pulumi config get <key>` command as shown below:
+
+```bash
+$ pulumi config get myEnvironment
+development
+```
+
+## Create secret values
+
+Pulumi Cloud always transmits and stores entire state files securely. Additionally, Pulumi supports encrypting specific values as “secrets” for extra protection. Encryption ensures that these values never appear as plain-text in your state file. By default, the encryption method uses automatic, per-stack encryption keys provided by Pulumi Cloud, but you can also use a [provider of your own choosing](/docs/concepts/secrets/#configuring-secrets-encryption) instead.
```

Comment:
Minor rephrase: "Pulumi supports encrypting specific values as “secrets” for extra protection. Pulumi Cloud transmits and stores state files over a secure connection, but once received the files are viewable in plain-text. By encrypting secrets, Pulumi ensures that these values never appear as plain-text in your state file. By default, the encryption method uses automatic, per-stack encryption keys provided by Pulumi Cloud, but you can also use a [provider of your own choosing](/docs/concepts/secrets/#configuring-secrets-encryption) instead."